uint8  id
string  poi_name
